src/java/utils/JsonHelper.java

```diff
@@ -26,6 +26,13 @@ public static JSONObject buildLanguageMap(String language, String label)
         return languageMap;
     }
     
+        public static JSONObject buildLanguageMapOtherContent(String language, String label)
```

Comment:
You do not need this function.  There already exists a helper function that does what you need (build a language map with `none`